src/components/_LuxUniversityAccessibility.vue

```diff
@@ -81,7 +79,6 @@ export default {
     a {
       color: var(--color-rich-black);
       background: var(--color-white);
-      &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}